packages/lib-user/src/components/shared/ProfileHeader/ProfileHeader.js

```diff
@@ -16,13 +16,17 @@ function TitledStat ({
   title = '',
   value = 0 
 }) {
+  let displayValue = value
+  if (isNaN(value)) {
+    displayValue = 0
+  }
```

Comment:
In what scenario could this be `NaN`? I ask because the `value` prop passed to TitledStat is a required `number`, so without an inline code comment here I'm confused why these additional lines are needed.

Reply:
Yes, great question, the reason these lines I think are necessary surprised me. `typeof NaN` is actually `'number'` (which made me lol, I had forgotten that JavaScript quirk). `NaN` can result from invalid arithmetic operations (as well as other reasons), and I think while loading stats or if there are no hours then the result of [the equation that converts the seconds](https://github.com/zooniverse/front-end-monorepo/pull/5926/files#diff-cb902d8eae8db0c3768ef71dfe357006cc3ba383ba8f774b32e29cda98161a08R31) can result in `NaN`.